vspace/src/main/java/edu/asu/diging/vspace/web/staff/UpdateStaffSpaceOrderModeController.java

```diff
@@ -0,0 +1,34 @@
+package edu.asu.diging.vspace.web.staff;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.vspace.core.model.ExhibitionSpaceOrderMode;
+import edu.asu.diging.vspace.core.services.IExhibitionManager;
+import edu.asu.diging.vspace.core.services.ISpacesCustomOrderManager;
+
+@Controller
+public class UpdateStaffSpaceOrderModeController {
```

Comment:
why is this called `...StaffSpace...` not just `...Space...`? There is not equivalent for the public page, right? This controller does not need to be contrasted with another one for the public site?